rewrite-core/src/main/java/org/openrewrite/internal/NameCaseConvention.java

```diff
@@ -201,7 +215,8 @@ private static String nameCaseJoiner(String str, boolean lowerCaseFirstLetter, c
                         builder.append(lowerCaseChar);
                     } else if (Character.isUpperCase(last) || last == '.') {
                         builder.append(lowerCaseChar);
-                    } else if (Character.isDigit(last) && (Character.isUpperCase(secondLast) || secondLast == separatorChar)) {
+                    } else if (Character.isDigit(last) && (Character.isUpperCase(secondLast)
+                                                           || secondLast == separatorChar)) {
```

Comment:
I would not reformat code.

Reply:
Sorry for that unrelated work. Reverted it 🙇